modules/index-service/src/main/java/org/opencastproject/index/service/resources/list/provider/EventsListProvider.java

```diff
@@ -117,15 +117,15 @@ public void setIndex(EventIndex index) {
       }
     } else if (COMMENTS.equals(listName)) {
       for (Comments comments : Comments.values())
-        list.put(comments.toString(), comments.toString());
+        list.put(comments.toString(), "EVENTS.EVENTS.DETAILS.COMMENTS." + comments.toString());
```

Comment:
Wouldn't it make more sense to use the FILTERS.EVENTS.COMMENTS translation keys here, since that's what this list provider is providing for?

Reply:
Wow, I must have lost orientation in the language file. I'll address this shortly.